src/api/api.ts

```diff
@@ -0,0 +1,35 @@
+export interface imageApiModel {
+    imageURL:string;
+    camera:string;
+    rover:string;
+    landingDate:string;
+    launchDate:string;
+    earthDate:string;
+}
+
+export async function imageAPI (date:string,rover:string,camera:string) {
+
+    let url = `https://api.nasa.gov/mars-photos/api/v1/rovers/${rover}/photos?${camera}&${date}api_key=jPTvIJZBife9yXLo9XXW1vDRXpKunAZUdbEGn5LM`;
```

Comment:
Can we use const here instead as this is a constant?
In the query string we need to make sure we set the param value correctly and use `&` between each query